README.md

```diff
@@ -119,8 +119,10 @@ The built-in blocks provided by Protoflow are listed below:
 | [`ReadDir`]     | Reads file names from a file system directory.             |
 | [`ReadEnv`]     | Reads the value of an environment variable.                |
 | [`ReadFile`]    | Reads bytes from the contents of a file.                   |
+| [`ReadSocket`]  | Reads proto message from a TCP port.                       |
```

Comment:
Let's change all instances of "proto message" to "Protocol Buffers message" as used elsewhere in the README.

Reply:
converted to 'Reads bytes from a TCP socket'